alchemy/src/cloudflare/bound.ts

```diff
@@ -50,31 +51,33 @@ export type Bound<T extends Binding> = T extends _DurableObjectNamespace<
                     ? Workflow<P>
                     : T extends D1DatabaseResource
                       ? D1Database
-                      : T extends _VectorizeIndex
-                        ? VectorizeIndex
-                        : T extends _Queue<infer Body>
-                          ? Queue<Body>
-                          : T extends _SecretsStore<infer S>
-                            ? SecretsStoreBinding<S>
-                            : T extends _AnalyticsEngineDataset
-                              ? AnalyticsEngineDataset
-                              : T extends _Pipeline<infer R>
-                                ? Pipeline<R>
-                                : T extends string
-                                  ? string
-                                  : T extends BrowserRendering
-                                    ? Fetcher
-                                    : T extends _Ai<infer M>
-                                      ? Ai<M>
-                                      : T extends _Images
-                                        ? ImagesBinding
-                                        : T extends _VersionMetadata
-                                          ? WorkerVersionMetadata
-                                          : T extends Self
-                                            ? Service
-                                            : T extends Json<infer T>
-                                              ? T
-                                              : Service;
+                      : T extends DispatchNamespaceResource
+                        ? Service
```

Comment:
There's a specific type for the bound dispatch namespace with get(name: string): Fetcher

see https://developers.cloudflare.com/cloudflare-for-platforms/workers-for-platforms/get-started/dynamic-dispatch/